book/security.rst

```diff
@@ -1352,6 +1352,15 @@ FriendsOfPHP organization.
     any of your dependencies is affected by a known security vulnerability.
     Therefore, you can easily integrate it in your build process.
 
+.. note::
+
+    To enable the ``security:check`` command, you need to add the
+    `sensio distribution bundle`_ in your composer dependencies.
```

Comment:
And by the way, it's only lowercased in the reference mapping at the bottom. In the text, it is written as "FrameworkExtraBundle documentation" (see http://symfony.com/doc/current/book/security.html#securing-controllers-and-other-code).